dataAccessObjects.py

```diff
@@ -159,6 +159,11 @@ def updateUserData(self,userInfo):
         sql = sql + 'user_level=50 WHERE user_name="{userName}"'.format(userName=str(userInfo["bluetooth_id"]))
         self.db.cmd(sql)
 
+    def updatePassword(self,hashedKey,userId):
+        sql = 'UPDATE user SET user_password="{hashedKey}" WHERE user_id="{userId}"'.format(
+                hashedKey=hashedKey,userId=userId)
```

Comment:
There is a table with all info on our google drive. Or just use the query `SHOW FIELDS FROM tablename`  to check.